### DISCUSSION TITLE
rewatch

### DISCUSSION THREAD

**Discussion by thekspr** [trakt/showly#673] [Ideas / Feature Requests]
Is it possible to add an option that the episode has been watched again? I saw that there is such an option on trakt.tv so it shouldn't be difficult, I hope. Because that's the only option I'm missing. 
I saw that someone already asked about this, but I'm curious if you've been working on it or if you're considering it at all

**Comment by michaldrabik** [comment]
Hi. Not working on it currently but I know that this feature is very much requested along with option to set custom watch date.

I know I gotta add those eventually although currently suffering from not much free time. It's not that simple though as multiple plays mean dealing with huge history collection so It need to be thought carefully.
